--- a/kernel/supercall/dispatch.c
+++ b/kernel/supercall/dispatch.c
@@ -1017,6 +1017,12 @@
     case CMD_SUSFS_ADD_SUS_KSTAT_STATICALLY: {
         susfs_add_sus_kstat(arg);
         return 0;
     }
+#ifdef CONFIG_KSU_SUSFS_SUS_KSTAT_REDIRECT
+    case CMD_SUSFS_ADD_SUS_KSTAT_REDIRECT: {
+        susfs_add_sus_kstat(arg);
+        return 0;
+    }
+#endif //#ifdef CONFIG_KSU_SUSFS_SUS_KSTAT_REDIRECT
 #endif //#ifdef CONFIG_KSU_SUSFS_SUS_KSTAT
 #ifdef CONFIG_KSU_SUSFS_SPOOF_UNAME
     case CMD_SUSFS_SET_UNAME: {
